app/controllers/events_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  before_action :require_user, except: [:index]
+
+  def new
+    @users = User.all
+    @users = current_user.except_current_user(@users).pluck(:name)
```

Comment:
If this is being required by your `new` and/or `show` methods, and any other method in the future, you might want to consider adding it as a callback.

[`OPTIONAL`]

Reply:
Furthermore, this `@users` instance variable that you are creating here can be extracted and placed in the `events_helper.rb` OR `users_helper.rb`. You are using this in your `new.html.erb` but it really isn't essential to your `new` or `create` actions, is it? I don't think it is because your `create_attendees` fetches the attendees from the form, and not from your `@users` instance variable, so this method is really just dependant on the information provided in the form; therefore, I'd strongly recommend moving this into its own method in a helper. 

[`OPTIONAL`]